Source/WebInspectorUI/UserInterface/Models/DOMNodeStyles.js

```diff
@@ -519,7 +519,7 @@ WI.DOMNodeStyles = class DOMNodeStyles extends WI.Object
     {
         var text = payload.text || "";
         var name = payload.name;
-        var value = payload.value || "";
+        var value = (payload.value || "").replace(/\s*!important\s*$/, "");
```

Comment:
There's no need to modify the value here in the parsing stage when receiving it from the backend. 
This is causing the tests to fail.

Reply:
Removed it